rds/sql/dialect/postgreSQL.ts

```diff
@@ -252,6 +257,11 @@ export abstract class AbstractPostgreSqlDialect extends diaImpl.ANSI {
         END;
       $BODY$;
       -- ALTER FUNCTION ${objWfDefnName}${argTypes} OWNER TO (TODO: OWNER);
+      INSERT INTO plpgsql_check_result
+      (function_id, line_no, "statement", sql_state, message, detail, hint,
+       "level", "position", query, context)
```

Comment:
@Kunjahamed-P @divyaSkumarCI01188 please check semicolon at end of insert; without it, the code will break?

Reply:
@shah 
We have already added [semicolon at the end of the insert](https://github.com/gov-suite/governed-im-transform/blob/master/rds/sql/dialect/postgreSQL.ts#L264).